brave/brave-ios
# Fix #6011: Suppress 'switch to app' annoyances by default.

## Summary of Changes


This pull request fixes #6011 

## Submitter Checklist:


## Test Plan:

1. Fresh install
2. Go to settings -> Brave Shields & Privacy, verify that 'Block 'Switch to App' Notices` is enabled by default.
3. Go to reddit.com, verify that there's no 'switch to mobiel app' showing
4. Disable the setting, go to reddit again and see if the popup is visible

## Screenshots:



## Reviewer Checklist:

  - `QA/(Yes|No)`
  - `bug` / `enhancement`
